### DISCUSSION TITLE
Case Sensitivity in Table names

### DISCUSSION THREAD

**Discussion by Kashinathpat** [sqlalchemy/sqlalchemy#11075] [Usage Questions]
I have one SQLite file and one MySQL Database 

1. MySQL Database has table names in LowerCase
2. SQLite has table names in CamelCase

- If I used `__tablename__` in lowercase SQLite throw error `sqlalchemy.exc.InvalidRequestError: Could not reflect: requested table(s) not available in Engine`
- And if I use CamelCase MySQL throws error

I want to use [this](https://github.com/rcw-2/python-digikamdb) but DigiKam uses CamelCase for Database Table name and MySQL use lowercase.

How do I make SQLAlchemy table names case-insensitive?

**Comment by CaselIT** [comment]
Hi,

I'm not sure this is supposed since it must be the db to be case insensitive, not sqlalchemy.

You can try using https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.quoted_name to see if you find a casing that works in both dbs.

Overall if tou plan on using the same tables/models in the two dbs then the tables should have the same name

**Reply by Kashinathpat** [reply]
> I'm not sure this is supposed since it must be the db to be case insensitive, not sqlalchemy.

I do Understand that I want to know if there is a way to ignore case-sensitivity in table names


> Overall if tou plan on using the same tables/models in the two dbs then the tables should have the same name

tables are generated by DigiKam software, their table names are same but MySQL doesn't support any uppercase character in tablename so they convert into lowercase
